internal/file/file_operator_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+package file
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/nginx/agent/v3/test/helpers"
+
+	mpi "github.com/nginx/agent/v3/api/grpc/mpi/v1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFileOperator_Write(t *testing.T) {
+	ctx := context.Background()
+
+	tempDir := t.TempDir()
+	filePath := filepath.Join(tempDir, "nginx.conf")
+	fileContent := []byte("location /test {\n    return 200 \"Test location\\n\";\n}")
```

Comment:
I think we should centralize the nginx configurations for testing purposes rather than have embedded strings everywhere